error: Jul-26-2022 21:11:25: 	uncaughtException: Cannot find module './back_end/routers'
Require stack:
- D:\medleymed\medleybackend\app.js
Error: Cannot find module './back_end/routers'
Require stack:
- D:\medleymed\medleybackend\app.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:885:15)
    at Function.Module._load (internal/modules/cjs/loader.js:730:27)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\medleymed\medleybackend\app.js:30:1)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
error: Jul-26-2022 21:23:24: 	uncaughtException: Invalid URL: mongodb://localhost:270171
TypeError [ERR_INVALID_URL]: Invalid URL: mongodb://localhost:270171
    at onParseError (internal/url.js:259:9)
    at new URL (internal/url.js:335:5)
    at new HostAddress (D:\medleymed\medleybackend\node_modules\mongodb\lib\utils.js:937:36)
    at fromString (D:\medleymed\medleybackend\node_modules\mongodb\lib\utils.js:987:16)
    at Array.map (<anonymous>)
    at parseOptions (D:\medleymed\medleybackend\node_modules\mongodb\lib\connection_string.js:218:45)
    at new MongoClient (D:\medleymed\medleybackend\node_modules\mongodb\lib\mongo_client.js:62:63)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:798:16
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:795:19)
error: Jul-26-2022 22:47:13: 	uncaughtException: verifyToken is not defined
ReferenceError: verifyToken is not defined
    at module.exports (D:\medleymed\medleybackend\back_end\routers\UserRoutes\user_roures.js:5:30)
    at module.exports (D:\medleymed\medleybackend\back_end\routers\UserRoutes\index.js:4:29)
    at module.exports (D:\medleymed\medleybackend\back_end\routers\index.js:4:28)
    at Object.<anonymous> (D:\medleymed\medleybackend\app.js:30:30)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47
error: Jul-26-2022 23:08:56: 	user_login.controller - insert - lineno-33, Error: ReferenceError: reqBody is not defined
error: Jul-26-2022 23:09:55: 	user_login.controller - insert - lineno-33, Error: ReferenceError: collectionName is not defined
error: Jul-26-2022 23:09:57: 	user_login.controller - insert - lineno-33, Error: ReferenceError: collectionName is not defined
error: Jul-26-2022 23:11:07: 	user_login.controller - insert - lineno-33, Error: TypeError [ERR_INVALID_ARG_TYPE]: The "cipher" argument must be of type string. Received undefined
error: Jul-27-2022 10:56:24: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 10:59:21: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 11:11:34: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 11:20:17: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 11:31:13: 	uncaughtException: Unexpected token ')'
D:\medleymed\medleybackend\back_end\db\models\UserModels\user_models.js:199
    console.log("error.code..", error.code, error.keyValue)[0])
                                                              ^

SyntaxError: Unexpected token ')'
    at wrapSafe (internal/modules/cjs/loader.js:984:16)
    at Module._compile (internal/modules/cjs/loader.js:1032:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (D:\medleymed\medleybackend\back_end\controllers\user_controller.js:1:14)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
error: Jul-27-2022 11:49:55: 	uncaughtException: option usecreateindex is not supported
MongoParseError: option usecreateindex is not supported
    at parseOptions (D:\medleymed\medleybackend\node_modules\mongodb\lib\connection_string.js:290:15)
    at new MongoClient (D:\medleymed\medleybackend\node_modules\mongodb\lib\mongo_client.js:62:63)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:798:16
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:795:19)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:379:10
    at promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:1224:10)
    at Mongoose.connect (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:378:20)
    at Object.<anonymous> (D:\medleymed\medleybackend\back_end\db\connection.js:10:10)
error: Jul-27-2022 11:50:03: 	uncaughtException: option usecreateindex is not supported
MongoParseError: option usecreateindex is not supported
    at parseOptions (D:\medleymed\medleybackend\node_modules\mongodb\lib\connection_string.js:290:15)
    at new MongoClient (D:\medleymed\medleybackend\node_modules\mongodb\lib\mongo_client.js:62:63)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:798:16
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:795:19)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:379:10
    at promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:1224:10)
    at Mongoose.connect (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:378:20)
    at Object.<anonymous> (D:\medleymed\medleybackend\back_end\db\connection.js:10:10)
error: Jul-27-2022 11:50:49: 	uncaughtException: option usecreateindex is not supported
MongoParseError: option usecreateindex is not supported
    at parseOptions (D:\medleymed\medleybackend\node_modules\mongodb\lib\connection_string.js:290:15)
    at new MongoClient (D:\medleymed\medleybackend\node_modules\mongodb\lib\mongo_client.js:62:63)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:798:16
    at new Promise (<anonymous>)
    at NativeConnection.Connection.openUri (D:\medleymed\medleybackend\node_modules\mongoose\lib\connection.js:795:19)
    at D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:379:10
    at promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\helpers\promiseOrCallback.js:10:12)
    at Mongoose._promiseOrCallback (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:1224:10)
    at Mongoose.connect (D:\medleymed\medleybackend\node_modules\mongoose\lib\index.js:378:20)
    at Object.<anonymous> (D:\medleymed\medleybackend\back_end\db\connection.js:10:10)
error: Jul-27-2022 11:55:55: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 11:59:03: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:00:18: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:00:41: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:00:59: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:03:51: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:05:53: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:06:12: 	user_login.controller - insert - lineno-33, Error: Error: Email name already exist
error: Jul-27-2022 12:06:58: 	user_login.controller - insert - lineno-33, Error: Error: Email already exist
error: Jul-27-2022 12:07:06: 	user_login.controller - insert - lineno-33, Error: Error: Email already exist
error: Jul-27-2022 12:07:08: 	user_login.controller - insert - lineno-33, Error: Error: Email already exist
error: Jul-27-2022 12:07:14: 	user_login.controller - insert - lineno-33, Error: Error: Phone number already exist
error: Jul-27-2022 12:07:50: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 12:11:43: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 12:14:30: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 12:15:04: 	uncaughtException: Route.get() requires a callback function but got a [object Undefined]
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (D:\medleymed\medleybackend\node_modules\express\lib\router\route.js:211:15)
    at Function.proto.<computed> [as get] (D:\medleymed\medleybackend\node_modules\express\lib\router\index.js:521:19)
    at module.exports (D:\medleymed\medleybackend\back_end\routers\UserRoutes\user_roures.js:7:12)
    at module.exports (D:\medleymed\medleybackend\back_end\routers\UserRoutes\index.js:4:29)
    at module.exports (D:\medleymed\medleybackend\back_end\routers\index.js:4:28)
    at Object.<anonymous> (D:\medleymed\medleybackend\app.js:30:30)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
error: Jul-27-2022 13:24:08: 	user_login.controller - insert - lineno-33, Error: ValidationError: name: name contains at least 3 letters
error: Jul-27-2022 13:24:18: 	user_login.controller - insert - lineno-33, Error: Error: Email already exist
error: Jul-27-2022 13:24:28: 	user_login.controller - insert - lineno-33, Error: Error: Email already exist
error: Jul-27-2022 13:24:34: 	user_login.controller - insert - lineno-33, Error: Error: Phone number already exist
